src/OpenSage.FileFormats.Big/BigArchive.cs

```diff
@@ -4,13 +4,11 @@
 using System.IO;
 using System.Text;
 using System.Threading;
-using OpenSage.Data.Utilities.Extensions;
-using OpenSage.FileFormats;
 
-namespace OpenSage.Data.Big
+namespace OpenSage.FileFormats.Big
 {
     [DebuggerDisplay("Archive: {FilePath}")]
-    public class BigArchive : DisposableBase
+    public class BigArchive : IDisposable
```

Comment:
Why change this? Or did you make this change before you moved DisposableBase to OpenSage.Core...

Reply:
Yes, that was done before. Will revert this